modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteClusterSnapshotCheckWithIndexesTest.java

```diff
@@ -36,6 +38,12 @@
  * Cluster-wide snapshot test check command with indexes.
  */
 public class IgniteClusterSnapshotCheckWithIndexesTest extends AbstractSnapshotSelfTest {
+    /** Parameters. Encryption is not supported by snapshot validation. */
```

Comment:
What is result if we start validate snapshot with both encrypted and non-encrypted caches? Can we check only non-encrypted caches then?

Reply:
Validation of incrypted snapshots is out of scope of this ticked. Implemented in following #9313